service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -68,43 +84,39 @@ void shouldBuildNotificationTemplatesWithCompleteValues() throws Exception {
         List<String> ordersAndDirections = ImmutableList.of("Emergency protection order",
             "Contact with any named person");
         Map<String, Object> expectedHmctsParameters = ImmutableMap.<String, Object>builder()
+            .putAll(expectedSimilarParameters)
             .put("court", "Family Court")
-            .put("localAuthority", "Example Local Authority")
-            .put("dataPresent", "Yes")
-            .put("fullStop", "No")
-            .put("ordersAndDirections", ordersAndDirections)
-            .put("timeFramePresent", "Yes")
-            .put("timeFrameValue", "Same day")
-            .put("reference", "12345")
-            .put("caseUrl", "http://fake-url/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345")
+            .put(ORDERS_AND_DIRECTIONS, ordersAndDirections)
             .build();
 
         Map<String, Object> expectedCafcassParameters = ImmutableMap.<String, Object>builder()
```

Comment:
Move to sepearate private method perhaps

Reply:
its was moved into a separate variable `expectedSimilarParameters`